css/layout.css

```
@@ -32,3 +32,38 @@
     width: 50%;
   }
 }
+
+img {
+  width: 100%;
+}
+
+.title-text {
+  padding-top: 50px;
+  margin-top: 60px;
+}
+
+.title {
+  padding-top: 5px;
+  padding-right: 5rem;
+}
+
+.url {
+  padding-top: 5px;
+  padding-right: 5rem;
+}
```

Why not combine these into 1 ruleset?
```suggestion
.title, .url {
  padding-top: 5px;
  padding-right: 5rem;
}
```